tests/mamba/test_selective_state_update.py

```
@@ -0,0 +1,297 @@
+import numpy as np
+import pytest
+import torch
+
+import flashinfer
+
+from .selective_state_update_triton import selective_state_update_triton
+
+
+def create_test_inputs(
+    batch_size, nheads, dim, dstate, ngroups, input_dtype, weight_dtype, matrixA_dtype, z_none=True
```

Ruff (0.14.10)

Unused function argument: `matrixA_dtype`

(ARG001)